opennlp-tools/src/main/java/opennlp/tools/namefind/RegexNameFinder.java

```diff
@@ -178,18 +197,34 @@ public void clearAdaptiveData() {
     // nothing to clear
   }
 
+  /**
+   * @return Retrieves the {@link Pattern patterns} used.
+   */
+  // TODO Unclear what 'm' stands for: Discuss refactoring, deprecation, or removal of unused method.
   public Pattern[] getmPatterns() {
     return mPatterns;
   }
 
+  /**
+   * @param mPatterns The {@link Pattern patterns} to be set.
+   */
+  // TODO Unclear what 'm' stands for: Discuss refactoring, deprecation, or removal of unused method.
   public void setmPatterns(Pattern[] mPatterns) {
     this.mPatterns = mPatterns;
   }
 
+  /**
+   * @return Retrieves the type used.
+   */
+  // TODO Unclear what 's' stands for: Discuss refactoring, deprecation, or removal of unused method.
   public String getsType() {
```

Comment:
Now this one got me... strings/input? Probably better refactor or remove.

Reply:
This looks like type of opennlp.tools.util.Span.class

Reply:
@kinow @atarora What's your opinion in terms of 
"refactoring, deprecation, or removal" of this unused method?

Either
- (A)  refactor it to `getSpanType()`
- (B)  refactor it to `getSpanType()` and deprecate
- (C) leave method name unchanged and provide an explanation of "s" via JavaDoc text
- (D)  remove without deprecation period

Reply:
+1 to A or B (we've made a few binary backward incompatible changes anyway :shrug: )

Reply:
A or B

Reply:
Option A it is, will proceed with that.